js-sdk/integrations/react/translating.mdx

```diff
@@ -1,59 +1,60 @@
 ---
 id: translating
-title: Translating (React)
+title: Translating
 sidebar_label: Translating
-description: 'Learn about several ways how to translate a text using Tolgee React integration: T-component or useTranslate hook.'
+description: "Tolgee enables you to implement localization in your React application with ease. You can integrate Tolgee in your React application using Tolgee's SDK for React. Learn to impelement translation using the useTranslate hook and the T-component."
 ---
 
 import TFunctionHint from '../../shared/_TFunctionHint.mdx';
 import NamespacesHint from '../../shared/_NamespacesHint.mdx';
-import ExampleBanner from '../../shared/ExampleBanner';
 
-<ExampleBanner framework="React" />
```

Comment:
Links to the example application are quite important, could we keep it here somewhere?

Reply:
The example on the top take away the focus and feels like that is the first thing we want the users do. I have tried updating the component, and have also added it to the bottom of the pages. Let me know what you think. I am attaching the screenshots here.
![Translating _ Tolgee](https://github.com/tolgee/documentation/assets/18901032/acd58a4f-fb2b-4845-9296-a296e1f7088b)

![Installation _ Tolgee](https://github.com/tolgee/documentation/assets/18901032/4a9f7e78-3fc0-43c0-ab0b-cfc78fe956f6)

Reply:
Yeah, thats cool 👍

Reply:
Just please fix the build, I think there are just some broken links, you can find out, if you run `npm run build` locally.